labs/open-enterprise-agent.md

```diff
@@ -0,0 +1,98 @@
+
+- Ability to develop controllers using the Open Enterprise Agent API to implement business logic tailored to specific use cases.
+- Flexibility to develop controllers in any programming language.
+- Written in Scala, a powerful, object-oriented language with functional programming language features that are highly scalable, based on JVM
+- Support of did:prism, did:peer and other did-methods via a universal resolver
+- Support of JWT-VCs and Anoncreds credential formats
+- The Open Enterprise Node interfaces with the Cardano blockchain when required, ensuring high levels of security, scalability, and availability.
+- The PRISM Open Enterprise Agent is designed as a cloudagent that can be accessed from anywhere.
+
+In summary, the Open Enterprise Agent is a versatile and powerful software agent that provides a straightforward REST API for easy integration and supports industry-standard protocols, making it well-suited for a wide range of decentralized identity use cases.
+
+## Current Features
+
+### PLATFORM SUPPORT
+
+- Server
+- Kubernetes
+- Docker
+
+### AGENT TYPES
+
+- Issuer
+- Holder
+- Verifier
+
+### CREDENTIAL TYPES SUPPORTED
+
+- W3C JWT-VC
+- Anoncreds
+
+### DID METHODS SUPPORTED
+
+- did:prism
+- did:peer
+- Other did-methods can be verified via a universal resolver
+
+### COMMUNICATION SUPPORT
+
+- DIDCOMM V2
+
+### DIDCOMM PROTOCOLS SUPPORTED
+
+- [Mediator Coordinator](https://didcomm.org/mediator-coordination/2.0/)
+- [DIDComm V2 Messaging](https://identity.foundation/didcomm-messaging/spec)
+- [DIDComm V2 Issue Credential](https://github.com/decentralized-identity/waci-didcomm/tree/main/issue_credential)
+- [DIDComm V2 Present Proof](https://github.com/decentralized-identity/waci-didcomm/blob/main/present_proof/present-proof-v3.md)
+- [DIDComm V2 Report Problem](https://identity.foundation/didcomm-messaging/spec/#problem-reports)
+- [DIDComm V2 Routing Protocol](https://identity.foundation/didcomm-messaging/spec/#routing-protocol-20)
+
+### SUPPORTED ARIES RFC’s
+
+- [0023-did-exchange](https://github.com/hyperledger/aries-rfcs/tree/main/features/0023-did-exchange)
+- [0434-out-of-band-protocol](https://github.com/hyperledger/aries-rfcs/blob/main/features/0434-outofband/README.md)
+- [0453-issue-credential-protocol](https://github.com/hyperledger/aries-rfcs/tree/main/features/0453-issue-credential-v2)
+- [0453-present-proof-protocol](https://github.com/hyperledger/aries-rfcs/tree/main/features/0454-present-proof-v2)
+
+# Initial Committers
+- https://github.com/patlo-iog
+- https://github.com/davidpoltorak-io
+- https://github.com/antonbaliasnikov
+- https://github.com/FabioPinheiro
+- https://github.com/mineme0110
+- https://github.com/bvoiturier
+- https://github.com/yshyn-iohk
+- https://github.com/CryptoKnightIOG
+- https://github.com/shotexa
+- https://github.com/goncalo-frade-iohk
+- https://github.com/lohanspies
+
+# Initial Maintainers
```

Comment:
How are the initial maintainers different from the initial committers?

Reply:
Will these two sets have different permissions?

Reply:
Yes. Once the code-base is transitioned, we intend on moving towards a community-based governance model, and would like to ensure that the maintainer list ensure a balanced representation of the community we intend to serve. In short: most developers will become 'regular' contributors.

Reply:
What I am trying to figure out is what GitHub teams will exist for this lab and what [GitHub permissions](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization) will be required? Will there be one team or two teams? If two teams what GitHub permissions will each team need?

Reply:
Ah got it. I'd argue that there's 2 sets of permissions, maintain (for the maintainers) and write (for contributors)